workshops/ARC.md

```diff
@@ -44,3 +44,87 @@ for hacking.
 
 [portfolio]: portfolio/README.md
 [twilio]: twilio/README.md
+
+## Concept Arc
+
+_Note: This section is still a work in progress and will be continually
+updated_
+
+The following is a listing of Hack Club's workshops. It breaks down which
+computer science concepts will be used in which workshops.
+
+**Workshop I: Personal Website**
+
+- Writing HTML tags
+- HTML tags go in order
+
+**Workshop II: Twilio**
+
+- Strings
+- Variables can be used to remember basic types
+- String concatenation
+- String concatenation with variables
+- Functions can be used with inputs
+- Introduce the idea that code runs in order
+
+**Workshop III: To Be Decided**
+
+- Emphasize that code runs in order
+- Emphasize that string concatenation is a form of syntactic sugar
```

Comment:
Do people need to know this? I thought it was something which people just picked up.